pre_commit/main.py

```diff
@@ -147,6 +147,13 @@ def _add_run_options(parser: argparse.ArgumentParser) -> None:
             'the rewrite'
         ),
     )
+    parser.add_argument(
+        '--is-rebase',
+        help=(
+            'During a pre-rebase hook, specifies the command that invoked '
+            'the rebase'
+        ),
+    )
```

Comment:
this isn't right -- hook-impl should collect the 1 or 2 arguments from https://git-scm.com/docs/githooks#_pre_rebase

Reply:
Done!